server/src/resolvers.js

```diff
@@ -129,6 +129,15 @@ export const Resolvers = {
     createSchedule(_, args, ctx) {
       return scheduleHandler.createSchedule(_, args, ctx);
     },
+    createTimeSegment(_, args, ctx) {
+      return timeSegmentHandler.createTimeSegment(_, args, ctx);
```

Comment:
`_` is supposed to be unused. If the function you're calling doesn't need it don't pass it. If you want to pass it then name it something other than `_`

Reply:
I should have caught this in previous reviews...

Reply:
I just copied the code above, should we clean them all up?